atum/src/main/scala/za/co/absa/atum/core/ControlType.scala

```diff
@@ -23,12 +23,16 @@ object ControlType {
   case object AbsAggregatedTotal extends ControlType("absAggregatedTotal")
   case object HashCrc32 extends ControlType("hashCrc32")
 
-  val values = Seq(Count.value, DistinctCount.value, AggregatedTotal.value, AbsAggregatedTotal.value, HashCrc32.value)
+  val values = Seq(Count, DistinctCount, AggregatedTotal, AbsAggregatedTotal, HashCrc32)
+  val valueStrings = values.map(_.value)
 
-  def getNormalizedValue(input: String) = {
-    values.find(value => isControlMeasureTypeEqual(input, value)).getOrElse(input)
+  def getNormalizedStringValue(input: String): String = {
+    valueStrings.find(value => isControlMeasureTypeEqual(input, value)).getOrElse(input)
   }
 
+  def withValueName(s: String): ControlType = values.find(_.value.toString == s).getOrElse(
```

Comment:
This method is intented to mimic Enumeration's `withName` ([used to parse string to get the enum value](https://www.scala-lang.org/api/current/scala/Enumeration.html#withName(s:String):Enumeration.this.Value))

But here, instead of matching on the direct `toString` representation, it matches on `value.toString`, hence the change `withName` -> `withValueName`.